### ISSUE TITLE
Problem with Recount addon

### ISSUE THREAD

**Issue by rinaldus** [ArndtB/WoW-Addon-Updater#2]
Hello!
I have a problem with updating Recount addon. It displays dialog box with error "On Request of URL http://addons.curse.cursecdn.com/files/2349/558/Recount-v7.1.0d release.zip some error(s) occurred. Please check the URL." and then crashes. After this it always crashes when I try to update addons, without displaying any dialog boxes. I tried to delete and re-add addon, but with the same result. URL link is 100% valid and I don't have problems with other addons.
Console output:
```
[xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
addonupd: /var/tmp/portage/x11-libs/libX11-1.6.4/work/libX11-1.6.4/src/xcb_io.c:165: dequeue_pending_request: Проверочное утверждение «!xcb_xlib_unknown_req_in_deq» не выполнено.
Аварийный останов (стек памяти сброшен на диск)
```
I deleted this addon from database. Without Recount it works normally. But can you fix this problem, please?

**Comment by ArndtB** [comment]
Hello rinaldus,

thank you for this detailed problem report. It seems to be a problem with space characters in the Filename / Download-URL. Since yesterday my wife have the same problem with the addon "Classic Quest Log" too. The Filename here was "Classic Quest Log_1_2_8.zip", this contains space characters too. I will check the routines and try to provide an Update for that soon.

Kind regards

Arndt

**Comment by ArndtB** [comment]
Fixed with 1.3.0 today.
